### PR TITLE
Tv static back ground

### LINKED ISSUE #10: Background
- [ ] Add background img.

### PR DESCRIPTION
**Pull request by PJSalter**
- Styling for no signal TV background.
- class attribute for background container.
- button within image container.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -24,4 +57,41 @@ h1 {
 
 .tv {
   margin: 2em;
+    /* needed to be relative and contain z-index to show over the background colors */
```
Comment (mariaalouisaa) [inline style.css:60]: Really appreciate these extra notes 🙏

File: style.css [hunk 2]
```diff
@@ -11,9 +11,42 @@ Dark Grey (#474747)
...
+  position: absolute;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+}
+
+.no-signal:after {
+  bottom: 0;
+  height: 21.73913%;
+  background: linear-gradient(to right, #006c6b 0%, #005857 16.66667%, white 16.66667%, #fafafa 33.33333%, #001b75 33.33333%, #001761 50%, #6c6c6c 50%, #626262 66.66667%, #929292 66.66667%, #888888 83.33333%, #3a3a3a 83.33333%, #303030 100%);
```
Comment (mariaalouisaa) [inline style.css:45]: This CSS is impressive! 🌟
Just read up one the :before & :after, I've never used it before so it's nice to see it in action 🤗

**Review by mariaalouisaa — APPROVED** [review]
Impressive work!

File: style.css [hunk 3]
```diff
@@ -11,9 +11,42 @@ Dark Grey (#474747)
...
+  width: 100%;
+  height: 100%;
+  position: relative;
+  background: linear-gradient(to right, #002fc6 0%, #002bb2 14.28571%, #3a3a3a 14.28571%, #303030 28.57143%, #ff0afe 28.57143%, #f500f4 42.85714%, #6c6c6c 42.85714%, #626262 57.14286%, #0affd9 57.14286%, #00f5ce 71.42857%, #3a3a3a 71.42857%, #303030 85.71429%, white 85.71429%, #fafafa 100%);
+}
+
+.no-signal:before{
+    top: 0;
+    height: 68.47826%;
+    background: linear-gradient(to right, white 0%, #fafafa 14.28571%, #ffe60a 14.28571%, #f5dc00 28.57143%, #0affd9 28.57143%, #00f5ce 42.85714%, #10ea00 42.85714%, #0ed600 57.14286%, #ff0afe 57.14286%, #f500f4 71.42857%, #ed0014 71.42857%, #d90012 85.71429%, #002fc6 85.71429%, #002bb2 100%);
```
Comment (AsmahanM) [inline style.css:31]: This is really interesting! It's nice to see how you worked- looks really good.

**Review by AsmahanM — APPROVED** [review]
Really neat CSS- looks good to me.

**Pull request merged**